jwtAuth.go

```diff
@@ -199,3 +221,10 @@ func GetClaims(request *http.Request) *TokenClaims {
 	}
 	return nil
 }
+
+/*
+IsVerifyToken is true when the AUTHENTICATION_VERIFY_TOKEN_SIGNATURE is properly set
+*/
+func IsVerifyToken() bool {
```

Comment:
```suggestion
func ShouldVerifyToken() bool {
```
`Is` usually indicates a state (passive) whereas in your case you want to actually perform an action